Rewrite MTV section in CNV docs (#99589)
openshift/openshift-docs: modules/migrating-to-virt.adoc

[id="migrating-to-virt_{context}"]
= Migrating to {VirtProductName}

You can migrate virtual machines (VMs) from external providers, such as {vmw-first} and {rh-openstack-first}, to {VirtProductName}. To make this process easier and more efficient, Red{nbsp}Hat provides {mtv-first}.

{mtv-short} is an Operator that you can install from the OperatorHub in the {product-title} web console. This toolkit simplifies the migration process and provides a central point of management for migrating VMs.

The migration process with {mtv-short} involves several key steps:

* link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/{mtv-version}/html/installing_and_using_the_migration_toolkit_for_virtualization/installing-the-operator_mtv#installing-the-operator_mtv[Installing the {mtv-short} Operator]: Before you can use the toolkit, you must install the {mtv-short} Operator on your {product-title} cluster. The Operator provides all the necessary components for migration.

> * link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/{mtv-version}/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index%23creating-a-migration-plan[Creating a migration plan]: A migration plan defines the VMs to be migrated, the source and destination environments, and other migration settings. This plan helps to organize and automate the migration process.

Author: ```suggestion
* link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/{mtv-version}/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#creating-plan-wizard-290-vmware_vmware[Creating a migration plan]: A migration plan defines the VMs to be migrated, the source and destination environments, and other migration settings. This plan helps to organize and automate the migration process.
```

> * link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/{mtv-version}/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index%23creating-a-network-map#about-network-maps_vmware[Mapping networks] and link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/{mtv-version}/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index%23creating-a-network-map#about-storage-maps_vmware[mapping storage]: {mtv-short} helps you to map networks and storage from your source environment (for example, vSphere or RHV) to the {product-title} environment. This ensures that the migrated VMs have the correct network and storage configurations.

Reviewer: These three links (Creating a migration plan, Mapping networks, and mapping storage) don't work for me; they all point to the top of the page as if the links were all to https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.8/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index 

@anarnold97 Do the links work on your end?

> * Performing the migration: Once the migration plan is created and the networks and storage are mapped, you can perform the migration. {mtv-short} handles the transfer of VM data and configuration to the OpenShift cluster.

Author: ```suggestion
* Performing the migration: When you have created the migration plan and the networks and storage are mapped, you can perform the migration. {mtv-short} handles the transfer of VM data and configuration to the OpenShift cluster.
```
